rx/assets/scss/rx.scss

```diff
@@ -0,0 +1,14 @@
+@import '../../../_sass/lib/bourbon/_bourbon.scss';
+@import '../../../_sass/lib/neat/_neat.scss';
+
+@import '../../../_sass/_va-variables.scss';
+@import '../../../_sass/core/_variables.scss';
+@import '../../../_sass/core/_grid-settings.scss';
+@import '../../../_sass/core/_grid.scss';
```

Comment:
Do you need to check in something (a Gemfile?) to make sure these libraries get installed? Looks like the build is failing.